docs/get-started/installation-problems/angular.mdx

```diff
@@ -1,3 +1,8 @@
+- To make Storybook CLI use npm for installation, you can use the following command:
+  ```shell
+  npx storybook init --use-npm
+  ```
+
 - Add the `--type angular` flag to the installation command to set up Storybook manually:
```

Comment:
@sruthigithub if you wouldn't mind, we can rephrase this for clarity, We can probably go with:
````md
Storybook's CLI provides support for both [Yarn](https://yarnpkg.com/) and [npm](https://www.npmjs.com/) package managers. If you have Yarn installed in your environment but prefer to use npm as your default package manager add the `--use-npm` flag to your installation command. For example:

```shell
npx storybook init --use-npm
```
````

Reply:
Thanks for the suggestion. Updated the document.